tema1/index.html

```diff
@@ -0,0 +1,37 @@
+<!DOCTYPE html> ,<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Practise1</title>
+  </head>
+  <header>
+    <nav>
+      <ul>
+        <li><a href="bio.html">Bio</a></li>
+      </ul>
+    </nav>
+  </header>
+  <script src="script.js"></script>
+  <body>
+    <h1>Depre Ema</h1>
+    <h2>Despre mine</h2>
+    <p style="color: blue">
+      Lucrez in <b>transport</b> de multi ani si doresc sa invat ceva <i>nou</i>
+    </p>
+    <h2>Hobby-uri din pacate sunt <u>putine</u></h2>
```

Comment:
Componenta de header ar trebuii sa fie in interiorul tag-ului <body>, la fel cum ati facut si in fisierul reteta.html, deoarece o bare de navigare face parte din continutul "corpului" unei pagini html. 


```tsx
  <body>
    <header>
      <nav>
        <ul>
          <li><a href="bio.html">Bio</a></li>
        </ul>
      </nav>
    </header>
    <h1>Depre Ema</h1>
    <h2>Despre mine</h2>
    <p style="color: blue">
      Lucrez in <b>transport</b> de multi ani si doresc sa invat ceva <i>nou</i>
    </p>
    <h2>Hobby-uri din pacate sunt <u>putine</u></h2>
    .....
    </body>
```